twyla/kubedeploy/kubectl.py

```diff
@@ -0,0 +1,73 @@
+import json
+import functools
+import subprocess
+
+
+class KubectlCallFailed(Exception):
+    pass
+
+
+class Kubectl:
+    def __init__(self):
+        self.exe = 'kubectl'
+        self.args = [
+            'get',
+            'pods'
+        ]
+        self.namespace = None
+
+
+    def _make_command(self):
+        cmd = []
+        cmd.append(self.exe)
+
+        if self.namespace:
+            cmd.extend(['--namespace', self.namespace])
+
+        cmd.extend(self.args)
+
+        return cmd
+
+
+    def _call(self, command, expect_json=True):
+        try:
+            proc = subprocess.run(
+                command,
+                stdout=subprocess.PIPE,
+                stderr=subprocess.PIPE)
+            proc.check_returncode()
+        except subprocess.CalledProcessError:
+            raise KubectlCallFailed(proc.stderr)
+
+        if expect_json:
+            return json.loads(proc.stdout.decode('utf8'))
```

Comment:
It's a visually annoying that `proc.stdout.decode('utf8')` is repeated here and on the next line. Maybe write it like this?:
```
text = proc.stdout.decode('utf8')
if expect_json:
    return json.loads(text)
return text
```